src/components/AddressField.vue

```diff
@@ -75,13 +94,27 @@ export default {
 
 <template lang="pug">
   div.inline-div
-    q-icon( v-if="this.contract" class="far fa-file-alt q-pr-xs")
-      q-tooltip Contract
     //- router-link(:to="`/address/${this.address}`") {{ getDisplay() }}
     a(:href="`/address/${this.address}`") {{ getDisplay() }}
+    q-icon(v-if="this.copy" class="far fa-copy" @click.stop="copyAddress")
+      q-tooltip Copy address
 </template>
 
 <style lang='sass' scoped>
-.inline-div
-  display: inline
+    .inline-div
+        display: inline
+
+    .far.fa-copy
+        margin-top: -5px
+        margin-left: 5px
+        margin-right: 15px
+        cursor: pointer
+        color: #666666
+        &:hover
+            transform: scale(1.2)
+            color: black
+
+    .body--dark .far.fa-copy
+        color: #c3c3c3
```

Comment:
move color hex to  [global variables](https://github.com/telosnetwork/teloscan/blob/master/src/css/quasar.variables.sass) and reference as variable